SafeguardSessions/CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 ## [Unreleased]
 
+### Compatible SPS Versions
+
+* 7.3.0, 7.4.0
+
+### Breaking changes
+
+* Power BI Service support has been implemented and the order and type of the input parameters have changed. After the upgrade, the input parameters of previously saved .pbix files must be transformed accordingly to work with the new versions of One Identity Safeguard Power BI Connector and One Identity Safeguard Power BI Report Template.<br>

```

Comment:
As a general principle I would try to avoid mixing html syntax with markdown. It works but IMHO it is not as pleasant to read a text filled with html tags. (Now it's only one, but if we keep it, it says this is the way.)

I think we can write "Transformation*: ..." in a new line like below, it does not need to be at the same indentation level as the previous sentence. The table is not at that level either. :smile:

Reply:
Done.